### DISCUSSION TITLE
antd 4.17.4 版本升级到 4.24.12

### DISCUSSION THREAD

**Discussion by yuanliu147** [ant-design/ant-design#44615] [Q&A] [answered]
我想将当前项目antd版本 4.17.4 升级到 4.24.12。请问，这种小版本更新会有什么不兼容的问题吗？

**Comment by Wxh16144** [comment, marked as answer]
没有人保证业务代码有没有魔改或者强依赖某个版本，提供一个检查顺序吧

1. 先了解 [语义化版本说明](https://semver.org/lang/zh-CN/)， antd 遵循了这个规范
2. 其次看 [antd 更新日志](https://4x.ant.design/changelog-cn)
3. 最后审查源代码 [compare 4.17.4...4.24.12](https://github.com/ant-design/ant-design/compare/4.17.4...4.24.12)

**Reply by yuanliu147** [reply]
好的，感谢。
